### PR TITLE
Add a map to contribute page

### PR DESCRIPTION
**Pull request by arvindsv**
* Is toggled off by default, for now. Add a '?map' to the URL of /contribute for it to show.
* Had to change the map a bit to satisfy some organizations.
* Locations:
  - https://cdn.rawgit.com/DmitryBaranovskiy/raphael/v2.1.4/raphael-min.js
  - https://cdn.rawgit.com/neveldo/jQuery-Mapael/1.1.0/js/jquery.mapael.js
  - https://rawgit.com/neveldo/jQuery-Mapael/1.1.0/js/maps/world_countries.js
  - https://cdnjs.cloudflare.com/ajax/libs/jquery-mousewheel/3.1.12/jquery.mousewheel.min.js

### CODE REVIEW

File: source/assets/javascripts/all.js [hunk 1]
```diff
@@ -3,9 +3,15 @@
 //= require 'lib/_masonry.pkgd'
 //= require 'lib/_handlebars-v4.0.5'
 //= require 'lib/_ramda'
+//= require 'lib/_raphael-min.js'
+//= require 'lib/_jquery.mapael.js'
+//= require 'lib/_world_countries.js'
+//= require 'lib/_jquery.mousewheel.min.js'
```
Comment (ketan) [inline source/assets/javascripts/all.js]: This is going down a rabbithole if we're going to add all JS and CSS in all pages for something that is only needed in a single page and loading up all pages to make them render slower.

Can you make it a separate JS file and include it in `layout.erb` —

```
<%= auto_stylesheet_link_tag %>
<%= auto_javascript_link_tag %>
```

Comment (arvindsv) [inline source/assets/javascripts/all.js]: I considered it, and thought we didn't want to. It used to be like that earlier, but not nice and in layout. I'll do this.

**Comment by arvindsv** [comment]
Oh, also, this is just a quick draft map, than @naveenbhaskar will style sometime.

**Comment by arvindsv** [comment]
@ketan, how about this?

**Comment by ketan** [comment]
LGTM, can you replace the `.min` files with development versions, so we can debug it if necessary? Merge it yourself at the end :)

**Comment by arvindsv** [comment]
Done. Thanks!

**Pull request merged**